portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -14,13 +14,12 @@
 
 package com.google.sps.data;
 
-import java.util.ArrayList;
-import java.util.List;
+public final class Comment {
+    private final String comment;
+    private final Long timestamp;
 
-public class Comments {
-    private final List<String> comments = new ArrayList<>();
-
-    public void addComment(String comment) {
-        comments.add(comment);
+    public Comment (String comment, Long timestamp) {
```

Comment:
I would recommend documenting the unit of the timestamp, such as "timestampMillis" or "timestampMs" or "timestampMicros".

Another possible solution is using java.time.Instant. Not sure how client friendly it is, though.

Reply:
Done